src/utils/cache-call-options.ts

```diff
@@ -0,0 +1,29 @@
+import {CollectionTtl} from './collection-ttl';
+
+export interface SingleCallOptions {
```

Comment:
I'm not attached to any of the names in here if anyone has alternatives.

Reply:
NumberTtlCallOptions?

Reply:
i was hesitant to put the name `Ttl` in the name of any of these because part of the motivation for including these in the first place is that the options list might expand in the future to include stuff other than TTLs.

So my recommendation for this one would probably just be `ScalarCallOptions` but I'm also okay with your current name.